docs/core-concepts/dotnet-core.md

```diff
@@ -0,0 +1,145 @@
+
+Comparison with .NET Framework
+------------------------------
+
+The .NET platform was first announced by Microsoft in 2000 and then evolved from there. The .NET Framework has been the primary .NET product produced by Microsoft during that 15+ year span. 
+
+The major differences between .NET Core and the .NET Framework: 
+
+- **App-models** -- .NET Core does not support the .NET Framework app-models, in part because many of them are built on Windows technologies, such as WPF (built on top of DirectX). The console and ASP.NET Core app-models are supported by both .NET Core and .NET Framework. 
+- **APIs** -- .NET Core contains many of the same, but fewer, APIs as the .NET Framework, and with a different factoring (assembly names are different; type shape differs in key cases). These differences currently typically require changes to port source to .NET Core. We intend to make the .NET Core API the same as the .NET Framework BCL API over time.
+- **Subsystems** -- .NET Core implements a subset of the subsystems in the .NET Framework, with the goal of a simpler implementation and programming model. For example, Code Access Security is not supported, while  Reflection is supported.
+- **Platforms** -- The .NET Framework supports Windows and Windows Server while .NET Core also supports OS X and Linux.
+- **Open Source** -- .NET Core is open source, while a [read-only subset of the .NET Framework](https://github.com/microsoft/referencesource) is open source.
+
+While .NET Core is unique and has significant differences to the .NET Framework and other .NET platforms, it is straightforward to share code, using either source or binary sharing techniques. 
+
+Comparison with Mono
+--------------------
+
+[Mono](http://www.mono-project.com/) is the original cross-platform and open source .NET implementation, first shipping in 2004. It can be thought of as a community clone of the .NET Framework. The Mono project team relied on the open [.NET standards](https://github.com/dotnet/coreclr/blob/master/Documentation/project-docs/dotnet-standards.md) (notably ECMA 335) published by Microsoft in order to provide a compatible implementation.
+
+The major differences between .NET Core and Mono:
+
+- **App-models** -- Mono supports a subset of the .NET Framework app-models (for example, Windows Forms) and some additional ones (for example, [Xamarin.iOS](https://www.xamarin.com/platform)) through the Xamarin product. .NET Core doesn't support these.
+- **APIs** -- Mono supports a [large subset](http://docs.go-mono.com/?link=root%3a%2fclasslib) of the .NET Framework APIs, using the same assembly names and factoring. .NET Core will support this same subset over time.
+- **Platforms** -- Mono supports many platforms and CPUs.
+- **Open Source** -- Mono and .NET Core both use the MIT license and are .NET Foundation projects.
+- **Focus** -- The primary focus of Mono in recent years is mobile platforms, while .NET Core is focussed on cloud workloads.
```

Comment:
Though I know we want to position it as such, given Immo's Slack stuff, not sure if this is true anymore.